front-end/cypress/e2e-extended/contacts/contacts.list.cy.ts

```diff
@@ -78,7 +78,7 @@ describe('Contacts List (/contacts)', () => {
     cy.contains('Save').should('not.exist');
     ContactListPage.goToPage();
     ContactsHelpers.assertColumnHeaders(ContactsHelpers.CONTACTS_HEADERS);
-    cy.contains('button,a', 'Restore deleted contacts').should('exist');
+    cy.contains('button,a', 'Restore deleted contacts').should('not.exist');
```

Comment:
What prompted changes to this test?  This one in particular seems to reverse the expected behavior?